test/jvm/src/test/scala/monocle/PlatformSpecificTestInstances.scala

```diff
@@ -5,7 +5,9 @@ import org.scalacheck.{Arbitrary, Cogen, Gen}
 import cats.Eq
 
 private [monocle] trait PlatformSpecificTestInstances {
-  implicit val urlEqual = Eq.fromUniversalEquals[URL]
+  implicit val urlEqual: Eq[URL] = Eq.instance {
```

Comment:
This `Eq` instance is bad as `equals` for `java.net.URL` is well known as blocking. It will try to do a dns lookup on the url making it very slow and it is also side effecting. The proper way should be to use URI instead

Reply:
ah good point, I learn that recently from john degoes tweet. I didn't realise we were using it in Monocle